docs/eventing/sources/sinkbinding/README.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "SinkBinding"
+weight: 01
+type: "docs"
+aliases:
+  - /docs/eventing/samples/sinkbinding/
+  - /docs/eventing/sources/sinkbinding
+---
+
+![API version v1](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
+
+The `SinkBinding` object supports decoupling event production from
+delivery addressing.
```

Comment:
as per #3650 - make the API objects formatting consistent, e.g. either `SinkBinding` or SinkBinding, either `Channel` or Channel

Reply:
I have decided to remove the code formatting. However, we should make a decision and add this to the style guide as this is not the only topic that is inconsistent.